is_prime.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(a):
+    i = 2
+
+    while i < a:
+        if a % i == 0:
+            return False
+        i += 1
+
+    return True
+
+
+
+print(is_prime(int(input("Enter number to check "))))
```

Comment:
Please make more readable by splitting into multiple lines

Reply:
Please answer to my comments so that I see that you've changed something

Reply:
I'll resolve the conversations after you comment that the issue is fixed. When all the conversations will get resolved we'll merge the PR into master